commands/games/trivia.js

```diff
@@ -0,0 +1,153 @@
+const {
+  SlashCommandBuilder,
+  EmbedBuilder,
+  ActionRowBuilder,
+  ButtonBuilder,
+  ButtonStyle,
+  ComponentType,
+} = require("discord.js");
+
+const { sleep } = require("../../utils/scripts/util_scripts.js");
+const questions = require("../../utils/json/trivia-questions.json");
+
+module.exports = {
+  //MARK: command data
+  data: new SlashCommandBuilder()
+    .setName("trivia")
+    .setDescription("Gives a random trivia question!"),
+  async execute(interaction, { embedColors }) {
+    const question = questions[Math.floor(Math.random() * questions.length)];
+    let questionText = `**${question.question}**\n`;
+
+    let peopleGuessed = 0;
+    let remainingTime = 15;
+    const questionEmbed = new EmbedBuilder()
+      .setTitle(questionText)
+      .setFooter({
+        text: `${peopleGuessed} guessed ● ${remainingTime} seconds left`,
+      })
+      .setColor(embedColors.blue);
+
+    const row = new ActionRowBuilder()
+      .addComponents(
+        new ButtonBuilder()
+          .setCustomId("0")
+          .setStyle(ButtonStyle.Primary)
+          .setLabel(question.options[0])
+      )
+      .addComponents(
+        new ButtonBuilder()
+          .setCustomId("1")
+          .setStyle(ButtonStyle.Primary)
+          .setLabel(question.options[1])
+      )
+      .addComponents(
+        new ButtonBuilder()
+          .setCustomId("2")
+          .setStyle(ButtonStyle.Primary)
+          .setLabel(question.options[2])
+      )
+      .addComponents(
+        new ButtonBuilder()
+          .setCustomId("3")
+          .setStyle(ButtonStyle.Primary)
+          .setLabel(question.options[3])
+      );
```

Comment:
perhaps randomize answer order? makes memorization slightly harder

Reply:
you could also make it have more than 4 options per trivia question with the same tech i used in flags, but idk